nextcloud/server
# fix(login): Disable login button when already logging in

## Summary

Fixes "State token does not match" in some cases (e.g. double click on the login button)

## How to reproduce:

* Start a login flow (V1 or V2), e.g. by logging in via the desktop client
* Enter username and password
* Click "Log in" and when the button changes to "Logging in" click the button again
* You're now prompted with a "temporary error". 
* Enter your password
* See the error "State token does not match"

## Before

<img width="410" alt="image" src="https://github.com/nextcloud/server/assets/1580193/5a3bb51e-4e7f-4644-b7a8-ae2f9131085f">


## After

<img width="444" alt="image" src="https://github.com/nextcloud/server/assets/1580193/401d5694-f836-44c1-95dd-2bfb0ae064a8">


## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
